kadasrouting/valhalla/connectors.py

```diff
@@ -48,12 +51,29 @@ def prepareMapmatchingParameters(self, shape, profile, options):
 
 
 class ConsoleConnector(Connector):
+    @staticmethod
+    def createValhallaJsonConfig(content):
+        # FIXME: this comes from a global variable
+        if not os.path.exists(APPDATA):
+            os.makedirs(APPDATA)
+        outputFileName = os.path.join(APPDATA, 'valhalla.json')
+        templatePath = os.path.join(os.path.dirname(os.path.dirname(__file__)), "valhalla")
+        templateFileLoader = FileSystemLoader(templatePath)
+        jinjaEnv = Environment(loader=templateFileLoader)
+        valhallaConfigTemplate = jinjaEnv.get_template('valhalla.json.jinja')
+        with open(outputFileName, 'w') as f:
+            f.write(valhallaConfigTemplate.render(valhallaTilesDir=content['valhallaTilesDir']))
+        return outputFileName
+
     def _execute(self, action, request):
-        valhallaPath = os.path.join(os.path.dirname(
-            os.path.dirname(__file__)), "executables", "valhalla")
+        defaultValhallaExeDir = 'C:\\Program Files\\KadasAlbireo\\opt\\routing'
```

Comment:
Probably better to use `os.environ['PROGRAMFILES']` instead of hardcoding the path to program files